route/index.js

```diff
@@ -139,3 +145,29 @@ const displayRouteData = data => {
   // the total amount of CO2 which were used with a petrol vehicle
   document.getElementById('co2').innerHTML = data.saving?.co2 ? `${data.saving.co2 / 1000} Kg` : 'Unknown';
 };
+
+const getGoogleMapDirectionsURL = legs => {
+  if (legs.length === 0) return;
+
+  let googleDirURL = `https://google.com/maps/dir/?api=1`;
+  const origin = legs[0].origin?.geometry?.coordinates;
+  const destination = legs[legs.length - 1].destination?.geometry?.coordinates;
+
+  // coordinates is an array with longitude as first value and latitude as second one
```

Comment:
coordinates are an array with longitude as first value and latitude as the second one

Reply:
:thumbUp: